src/App.js

```diff
@@ -1,22 +1,14 @@
-import logo from './logo.svg';
 import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Greeting from './Components/Greeting';
 
 function App() {
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <Routes>
+          <Route path="/" exact component={Greeting} />
+        </Routes>
       </header>
     </div>
   );
```

Comment:
Hi there,

I've noticed that you've made some changes to the `App.js` file to include Redux and React Router. These are great additions, but I have a few suggestions to improve the code:

1. **Provider Usage**: The `Provider` component from `react-redux` is usually should be added, and the store should be provided from the redux.

3. **BrowserRouter**: You need to add a `BrowserRouter` component, and you haven't imported it. Make sure to import it from `react-router-dom` at the top of your file, and use it or else you could have error with react routers.

Here's how I would suggest modifying your code:

```javascript
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import { Provider } from 'react-redux';
import './App.css';
import Greeting from './Components/Greeting';
import store from './redux/store';

function App() {
  return (
    <Provider store={store}>
      <BrowserRouter>
        <div className="App">
          <header className="App-header">
            <Routes>
              <Route path="/" exact component={Greeting} />
            </Routes>
          </header>
        </div>
      </BrowserRouter>
    </Provider>
  );
}

export default App;